ruby-challenges.rb

```diff
@@ -3,7 +3,22 @@
 
 # --------------------1) Create a method that takes in an array of words and a single letter and returns an array of all the words containing that particular letter. Use the test variables provided.
 
+letter_o = 'o'
+letter_t = 't'
+
 beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+def particular_letters(array, letter)
+    letter_result = []
+    
+    array.each do |array|
+        if array.include?(letter)
+            letter_result << array
+        end
+    end
+    return letter_result
+end
+p particular_letters(beverages_array(letter_o))
+p particular_letters(beverages_array(letter_t))
```

Comment:
Like in the JavaScript challenges, you are trying to perform a method call with `beverages_array`, which is not a method, so it will output an error.